agent/consul/internal_endpoint_test.go

```diff
@@ -1,9 +1,9 @@
 package consul
 
 import (
+	"crypto/rand"
 	"encoding/base64"
 	"fmt"
-	"math/rand"
```

Comment:
`math/rand.Read` is deprecated, but `crypto/rand.Read` has the same API, so it is a drop-in replacement everywhere.